en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+                   <idToDelete>{$ctx:idToDelete}</idToDelete>
+               </fhir.delete>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/update">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToUpdate)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.update>
+                   <type>{$ctx:type}</type>
+                   <idToUpdate>{$ctx:idToUpdate}</idToUpdate>
+                   <format>{$ctx:format}</format>
+               </fhir.update>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+
+    ```
+3. Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime.
+
+{! /references/connectors/exporting-artifacts.md !}
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+    
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here](https://curl.haxx.se/download.html).
+
+1. Create resource operation.
+   
+   **Sample Request** 
+      
+   ```
+   curl -v POST -d 
+   '{
+      "base": "http://hapi.fhir.org/baseR4",
+      "resourceType": "Patient",
+      "format": "json",
+      "name": [{"family": "Jhone","given": ["Winney","Rodrigo"]}]
+    }' "http://localhost:8290/resources/create" -H "Content-Type:application/json" 
+       
+   ```
+   **Expected Response**: 
+      
+   ```
+   <jsonObject>
+       <resourceType>Patient</resourceType>
+       <id>698021</id>
+        <meta>
+          <versionId>1</versionId>
+          <lastUpdated>2020-03-24T07:57:14.506+00:00</lastUpdated>
+        </meta>
+         <text>
+           <status>generated</status>
+           <div>&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;div class="hapiHeaderText"&gt;Winney Rodrigo &lt;b&gt;JHONE &lt;/b&gt;&lt;/div&gt;&lt;table class="hapiPropertyTable"&gt;&lt;tbody/&gt;&lt;/table&gt;&lt;/div&gt;</div>
+         </text>
+         <name>
+            <family>Jhone</family>
+            <given>Winney</given>
+            <given>Rodrigo</given>
+        </name>
+   </jsonObject>
+   ```  
+           
+2. readResource operation.
```

Comment:
Same here, we'll add what we are going to do

Reply:
Changed.